bccsp/schemes/dlog/crypto/signature.go

```diff
@@ -21,6 +21,7 @@ import (
 // signLabel is the label used in zero-knowledge proof (ZKP) to identify that this ZKP is a signature of knowledge
 const signLabel = "sign"
 const signWithEidNymLabel = "signWithEidNym"
+const signWithEidNymRhNymLabel = "signWithEidNymRhNym"
```

Comment:
Can you please clarify something: whenever a revocation handle is there, the nym eid must be there as well? Isn't there the case where we want to have the revocation handle but not the nym eid?

Reply:
From my understanding you would want both the revocation handle and enrollment id because there would always be an enrollment id and you'd want to make sure it's revoked

Reply:
Thx - just to double-check with @adecaro that this assumption holds. If it does, let's please add a comment about it in the code.

Reply:
yes, we want to have them both always.